src/components/PostDetails.tsx

```diff
@@ -114,17 +44,17 @@ export const PostDetails: React.FC<Props> = ({ post }) => {
           </div>
         )}
 
-        {loaded && !hasError && comments.length === 0 && (
+        {loaded && !hasError && items.length === 0 && (
```

Comment:
```suggestion
        {loaded && !hasError && !items.length && (
```

Reply:
3 minutes after I downloaded it, it still hasn't had time to download